src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  const indefityPerson = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {' '}
+        My name is
+        {name}
+      </h2>
+      {age && (
+      <p className="Person__age">
+        {' '}
+        I am
+        {age}
+      </p>
```

Comment:
I check linter settings for this task. You can return your previous solution with interpolation

Reply:
and add right space 
![image](https://github.com/mate-academy/react_person/assets/70009205/1892935b-b531-44cd-8dcb-b107f9820758)